bin/dump_data.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+# Dump component data
+# Run this script from the root of the repository
+# dump_data.sh for all apps or dump_data.sh zaken documenten ... to specify specific apps
+# Note that postgres 17 requires postgres-client-17
+#
+# By default a schema and data dump are created separately. This can be changed with the flags --data-only, --schema-only
+# or --combined which appends the data dump to the schema dump.
+# The schema dump could not use -t to filter tables because this excludes extensions like postgis in the dump.
+# pg_dump also does not add related tables automatically, so `dump_data.sh zaken` does not add related zaaktype data to the dump.
+
+
+set -e
+
+DEFAULT_APPS=(besluiten catalogi documenten zaken)
+
+export PGHOST=${DB_HOST:-db}
+export PGPORT=${DB_PORT:-5432}
+export PGUSER=${DB_USER:-postgres}
+export PGDATABASE=${DB_NAME:-postgres}
+export PGPASSWORD=${DB_PASSWORD:-""}
+
+SCRIPT=$(readlink -f "$0")
+SCRIPTPATH=$(dirname "$SCRIPT")
+
+${SCRIPTPATH}/wait_for_db.sh
+
+DUMP_FILE=${DUMP_FILE:-"dump_$(date +'%Y-%m-%d_%H-%M-%S').sql"}
+
+SCHEMA=true
```

Comment:
When you load the schema, note that the django_migrations table will still be empty, so if you actually deploy an instance things will crash, because the tables already exist. I think we should at least document this (that you should run `src/manage.py migrate --fake`) or maybe we should include django_migrations in the data dump as well to avoid this